File: go/divisors.go
package main

import (
	"fmt"
	"math"
)

func getDivisors(n int) int {
	if n == 1 {
		return 1
	}
	divisors := 1
	for i := 1; i <= int(math.Sqrt(float64(n))); i++ {
		if n%i == 0 {
			divisors++
		}
	}
	return divisors
}

// driver code
func main() {
	fmt.Printf("divisors: %d\n", getDivisors(4324320))
}

File: r/divisors.r
getDivisors <- function(n) {
    if (n == 1) {
        print(1)
    } else {
        divisors <- 0
        for (i in 1:sqrt(n)) {
            if (n %% i == 0) {
                divisors = divisors + 1
            }
        }
        print(divisors * 2)
    }
}

"driver code"
getDivisors(4324320)